[website] Update Careers page (#24948)
mui-org/material-ui: docs/src/pages/company/careers/careers.md

# We build the next generation of tools for UI development

<p class="description">Material-UI is focused on making React UI development faster, simpler, and accessible to more people.</p>

The open-source project started back in 2014 to unify React and Material Design. Today, Material-UI has grown to become one of the world's most popular React UI libraries, backed by a vibrant community of more than 2M developers in over 180 countries.

> The company is self-bootstrapped (so far). It was incorporated in late 2019 and yet growing fast (x2-3 YoY). We have doubled the team in 2020 and are on track to do the same in 2021 (planning to onboard six new members).

Reviewer: ```suggestion
The company is bootstrapped (so far). It was incorporated in mid 2019 and yet growing fast (x2-3 YoY). We doubled the team in 2020 and are on track to do the same in 2021.
```

## Our values

- **Customer obsessed**. We put our customers front & center. We focus on delivering web experiences that feel amazing. We ask questions. We experiment with new ideas.
- **Transparency**. Most of our work is public. We share what we work on regularly in the open. We collect feedback as soon and as frequently as possible. We learn from each other and
grow from it.
- **Freedom**. We work from anywhere in the world. We keep meetings to a minimum, preferring asynchronous and written communication.
- **Autonomy**. We want to create a safe, high-trust team. We want you to be able to feel invested in your work and proud of it.
- **Excellence**. We're aiming high, and we know it. Even the smallest details are important to us. We only want the best for each other and our customers.

## Perks & benefits

- **Remote**: Our entire company is distributed.

> - **Gatherings**: We meet up two times a year for a short week of meetings, events, and fun!

Reviewer: ```suggestion
- **Gatherings**: We meet up once or twice a year for a short week of meetings, events, and fun!
```

- **Equipment**: Material-UI will let you choose new hardware of your choice (up to $2,500 USD).
- **Flexible hours**: We work from different timezones. When the work is done doesn't really matter.

> - **Time off**: We provide 5 weeks if paid time off.

Reviewer 2: ```suggestion
- **Time off**: We provide 5 weeks of paid time off.
```

> - **Time off**: We provide 5 weeks of paid time off.

Reviewer: ```suggestion
- **Time off**: We provide five weeks of paid time off.
```

## Open roles

We're currently looking for help in the following areas:

### [Lead Designer](/company/lead-designer/)

We are looking for a lead designer to own most of our design work.

### [Senior Software Engineer](/company/software-engineer/)

> <span style="font-size: 14px;">x2 positions (1 design system team, 1 advanced components team)</span>

Reviewer: ```suggestion
<span style="font-size: 14px;">Two positions (one open-source team, one advanced components team).</span>
```

We are looking for senior software engineers to help support our open-source components, assist the Material-UI community, and grow our enterprise edition.

## FAQs

### Are there application deadlines?

No. If a job is visible on our careers page, then you can still apply.

### Does Material-UI do whiteboarding during interviews?

No. We ask applicants to complete challenges that are close to their future day-to-day contributions.

### Does Material-UI offer contractor job opportunities?

> Yes. People outside of France will be hired as full-time contractors.

Reviewer: ```suggestion
Yes. People outside of France will be hired as full-time contractors. (Benefits may not apply.)
```

Author: ```suggestion
Yes. People outside of France will be hired as full-time contractors. (Benefits may vary.)
```
